pkgs/development/libraries/cyrus-sasl/openssl3.patch

```diff
@@ -0,0 +1,196 @@
+From 455417ad5d7da87d22590942a433939bdff986ca Mon Sep 17 00:00:00 2001
```

Comment:
Upstream commit seems to be https://github.com/cyrusimap/cyrus-sasl/commit/887dbc0435056ec58ee48c4d803f110ade1e4c39

Reply:
It's three commits https://github.com/cyrusimap/cyrus-sasl/commit/887dbc0435056ec58ee48c4d803f110ade1e4c39, https://github.com/cyrusimap/cyrus-sasl/commit/dfaa62392e7caecc6ecf0097b4d73738ec4fc0a8 and https://github.com/cyrusimap/cyrus-sasl/commit/c2bd3afbca57f176d8c650670ce371444bb7fcc0

Reply:
https://patch-diff.githubusercontent.com/raw/cyrusimap/cyrus-sasl/pull/653.patch

You can do the same on a range of commits if you want to pin it.